### ISSUE TITLE
Fulfillment by own warehouse and fulfillment by Allegro

### ISSUE THREAD

**Issue by bartflow** [allegro/allegro-api#9935]
We have customers that want both ways, so we heard that you need 2 accounts, right?:

Do you get the same offer twice at Allegro? Does it work together or is it FBM or FBR and not both at the same time?

For example:

If the customer has their own stock and we send it as an offer, but Allegro also has a stock. And the question is, is this possible for them or are they against it? All other channels that support FBM always either do FBM or your own stock, but never both at the same time.

Can we at FBM control prices via the API? 
Can we pick up the FBM stock? 
Is it a different type of account? 
What if we accidentally send stock because a customer has logged in incorrectly or something like that?

**Comment by Lukasz-Zurek** [comment]
Yes. You should register One Fulfillment account - seller's account must be connected with One Fulfillment by Allegro Service. You can [read more](https://developer.allegro.pl/tutorials/one-fulfillment-by-allegro-4R9dXyMPlc9). So in this situation Allegro manages your orders automatically, but please read our tutorial step by step how it looks in our API. Not all of the products can be listed here.

You can also find some more details about this service right [here](https://allegro.pl/kampania/en/fulfillment#)

Another thing - you can register a different account (business account) where you can configure orders management - we described events and orders right [here](https://developer.allegro.pl/tutorials/process-orders-PgPMlWDr8Cv) and offers management as well - read more [here](https://developer.allegro.pl/tutorials/how-to-process-list-of-offers-m09BKA5v8H3). It's a complex process and in this situation - the configuration and your own warehouse management process is fully up to you.  We don't have dedicated resources for "fulfillment by merchant", etc. in our public API. Using our resources, you need to determine how this will be managed in the context of your warehouse..
